tabulon_dxf/src/lib.rs

```diff
@@ -381,7 +386,8 @@ pub fn load_file_default_layers(path: impl AsRef<Path>) -> DxfResult<TDDrawing>
                     ),
                     alignment: Default::default(),
                     insertion: DirectIsometry::new(
-                        mt.rotation_angle.to_radians(),
+                        // As far as I'm aware, x_axis_direction and rotation are exclusive.
```

Comment:
In a follow-up commit, it might be good to emit a notice via `tracing` if both are set. Although the user can't do much about that, so maybe at trace level.

Reply:
That's fair, I'll keep this in mind when we hook up tracing.